Leak in client-go/v2/txnkv/transaction.keepAlive

## Bug Report

Please answer these questions before submitting your issue. Thanks!
```
[2021-08-08T10:09:47.248Z]     leaktest.go:177: Test TestT check-count 50 appears to have leaked: github.com/tikv/client-go/v2/txnkv/transaction.keepAlive(0xc0fabea780, 0xc08e025680, 0xc08e12d05b, 0x1c, 0x1c, 0xc08e101320)
[2021-08-08T10:09:47.248Z]         	/nfs/cache/mod/github.com/tikv/client-go/v2@v2.0.0-alpha.0.20210806085616-14892a598eab/txnkv/transaction/2pc.go:897 +0x105
[2021-08-08T10:09:47.248Z]         created by github.com/tikv/client-go/v2/txnkv/transaction.(*ttlManager).run
[2021-08-08T10:09:47.248Z]         	/nfs/cache/mod/github.com/tikv/client-go/v2@v2.0.0-alpha.0.20210806085616-14892a598eab/txnkv/transaction/2pc.go:870 +0xfe
[2021-08-08T10:09:47.248Z] --- FAIL: TestT (152.43s)
[2021-08-08T10:09:47.248Z] FAIL
```

### 1. Minimal reproduce step (Required)
in ci https://ci.pingcap.net/blue/rest/organizations/jenkins/pipelines/tidb-unit-test-nightly/runs/7911/nodes/132/steps/544/log/?start=0

<!-- a step by step guide for reproducing the bug. -->

### 2. What did you expect to see? (Required)

### 3. What did you see instead (Required)

### 4. What is your TiDB version? (Required)

<!-- Paste the output of SELECT tidb_version() -->

--- a/ddl/db_test.go
+++ b/ddl/db_test.go
@@ -339,6 +339,11 @@ func (s *testSerialDBSuite) TestWriteReorgForColumnTypeChangeOnAmendTxn(c *C) {
 
 		var checkErr error
 		tk1 := testkit.NewTestKit(c, s.store)
+		defer func() {
+			if tk1.Se != nil {
+				tk1.Se.Close()
+			}
+		}()
 		hook := &ddl.TestDDLCallback{Do: s.dom}
 		times := 0
 		hook.OnJobUpdatedExported = func(job *model.Job) {

